databricks-sdk-java/src/test/java/com/databricks/sdk/core/ApiClientTest.java

```diff
@@ -100,6 +123,16 @@ private SuccessfulResponse getTooManyRequestsResponse(Request req) {
         new Response(req, 429, "Too Many Requests", Collections.emptyMap(), (String) null));
   }
 
+  private SuccessfulResponse getTooManyRequestsResponseWithRetryAfterHeader(Request req) {
+    return new SuccessfulResponse(
+        new Response(
+            req,
+            429,
+            "Too Many Requests",
+            Collections.singletonMap("retry-after", Collections.singletonList("1")),
```

Comment:
Can we add a case for retry-after with absolute date as well, if we're going to support it?